src/components/EBIHeader/style.css

```
@@ -1,25 +1,68 @@
-.masthead-black-bar a:visited {
-  color: white;
-}
-
 /* Overwrite EBI guidelines styles… */
-.global-nav > * {
-  float: initial !important;
+.masthead-black-bar {
+  --vf-stack-margin--custom: 0;
+  background-color: var(--vf-color--grey--darkest);
+  /* border-top: 0 solid #eee; */
+
+  & > nav {
+    max-width: var(--vf-body-width, 80em);
+    & .global-nav {
+      & > li {
+        font-size: 14.4px;
+        float: initial !important;
 
-  &.embl-selector {
-    float: right !important;
+        & > a {
+          &,
+          &:visited,
+          &:focus {
+            color: white;
+          }
+          &:not(.float-right):hover {
+            color: #373a36;
+            background-color: #d0d0ce !important;
```

I don't know why I originally didn't use the custom properties.

```suggestion
            color: var(--vf-color--grey--darkest);
            background-color: var(--vf-color--grey--lightest);
```